src/cosmic_ray/cli.py

```diff
@@ -96,7 +97,11 @@ def init(config_file, session_file):
             log.info(" - %s: %s", directory, ", ".join(sorted(files)))
 
     with use_db(session_file) as database:
-        cosmic_ray.commands.init(modules, database, operators_cfg)
+        if database.num_results>0:
```

Comment:
I think this logic is incorrect. If `database.num_results` is greater than zero, it will always exit, event if `force` is `True`. What we want is for initialization to occur if `force` is `True`, no matter what the state of the existing database is.